Scripts/symbolic_hermitians/HermitianUtils.py

```diff
@@ -70,16 +93,19 @@ def anticommutator(self, HermitianA, HermitianB):
         return self
 
     def conjugate(self):
-        self.H = Conjugate(self.H)
+        for i in range(self.size):
+            for j in range(i, self.size):
```

Comment:
Ah, you are totally correct - in many of the other loops we only do the upper half because that's all the code we generate or all we need to construct the matrix. But that is not the case here, so I'll fix it.